packages/roosterjs-editor-core/lib/corePlugins/ImageSelection.ts

```diff
@@ -43,7 +43,7 @@ export default class ImageSelection implements EditorPlugin {
                     if (
                         safeInstanceOf(target, 'HTMLImageElement') &&
                         target.isContentEditable &&
-                        event.rawEvent.button === mouseLeftButton
+                        event.rawEvent.button != mouseMiddleButton
```

Comment:
do we need this change?

Reply:
ImageSelection is broken on context menu without it